### DISCUSSION TITLE
'Undefined' error when I try to export from figma plugin

### DISCUSSION THREAD

**Discussion by gold-olar** [dicebear/dicebear#417] [Q&A] [answered]
![Screenshot 2024-06-10 at 2 31 53 AM](https://github.com/dicebear/dicebear/assets/43508719/fb2561f0-8aa2-4df2-abc0-b67ae144d49a)

I'm not sure what's going on here, but I keep getting this error when I try to export the library I created on figma. 
I had been exporting this library without issues until now, so i am not sure what has changed.

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @gold-olar 

To be able to help you, I need to debug this using your file. Would you like to send me the file to contact@dicebear.com?

**Reply by gold-olar** [reply]
> Hi @gold-olar
> 
> To be able to help you, I need to debug this using your file. Would you like to send me the file to [contact@dicebear.com](mailto:contact@dicebear.com)?

Thanks for the response. I was able to resolve this by disabling each asset and trying to export. I found the one causing the issue and I think i am all good now. thanks for your help.
